pkg/asset/manifests/dns.go

```diff
@@ -96,6 +98,14 @@ func (d *DNS) Generate(dependencies asset.Parents) error {
 		config.Spec.PrivateZone = &configv1.DNSZone{
 			ID: dnsConfig.GetDNSZoneID(clusterID.InfraID+"-rg", installConfig.Config.ClusterDomain()),
 		}
+	case gcptypes.Name:
+		zone, err := icgcp.GetPublicZone(installConfig.Config.BaseDomain, installConfig.Config.Platform.GCP.ProjectID)
```

Comment:
The public zone is based on the base domain. so that requires the lookup. so this is fine.